tinydoor/templates/pages/home.html

```diff
@@ -8,7 +8,15 @@
 {% endblock css %}
 
 {% block content %}
-<button id='browse'>Upload</button>
+  <p>Welcome to</p>
```

Comment:
Hey Ashab! This looks like a good start.

Did you mean to just put "Welcome to" here? It seems a little incomplete to me, please clarify.

Overall, I like how you have added code in a very modular way. You'll have to tell me more about ```.idea```, I've never seen this tool before!